src/components/popover/index.tsx

```diff
@@ -13,17 +13,33 @@ import {
 type PopoverProps = PropsWithChildren<
   {
     content: ReactNode;
+    trigger?: 'hover' | 'click';
+    contentWidth?: string;
```

Comment:
for width and padding, can we somehow solve it through variants instead of those open props?

Reply:
Yes, I created a size variant